### DISCUSSION TITLE
Is this allowed???

### DISCUSSION THREAD

**Discussion by pwrj** [AzuraCast/AzuraCast#3617] [Q&A] [answered]
Are people really selling Azuracast?? I stumbled up on this page: https://radiolize.com/pricing/ 
I couldn't believe it!!!

**Comment by BusterNeece** [comment, marked as answer]
@pwrj In this case, and in most cases of people out there "reselling" AzuraCast, they're actually just selling hosting services for AzuraCast rather than trying to sell the software itself. In either case, there isn't much we can do about it from a licensing perspective, as we are limited by the terms of our open-source license.

See #3301 for more info.

**Reply by pwrj** [reply]
@SlvrEagle23 It sure sounds unfair.

**Reply by BusterNeece** [reply]
It really depends on the reseller/host.

Sometimes, they do a great job hosting customers and making it easier for new users to get involved with AzuraCast, and they contribute back to us via donations and support us. Those resellers are a useful and helpful part of our ecosystem and we have no issues with them at all.

Others provide a sub-par experience to their customers, while not donating back to us and directing their users to ask us directly about any support questions, despite the fact that they're selling our software for a profit they don't share with us.

That second group is bad, bad enough to sometimes make me want to not do anything for free ever again.

**Reply by pwrj** [reply]
@SlvrEagle23 I hear you and it's disheartening.

**Reply by johnrisby** [reply]
@BusterNeece Do you know which camp Radiolize fall into?

**Reply by BusterNeece** [reply]
@johnrisby Radiolize forked AzuraCast back when it was licensed under the Apache 2.0 license, which allowed them to use it commercially and modify it however they pleased without sharing the source code back with the community. Since then, they've gone their own way and modified the code quite a bit to produce their Studio product.

License-wise and legally, they haven't done anything wrong. They do, however, in my opinion violate the spirit of the open-source community by building on top of FOSS software, in particular years and years of my own hard work, but not giving back to the FOSS ecosystem. Along with several other resellers, they are among the reasons we modified our license to be the AGPL, which *does* require that resellers modifying AzuraCast make the source code for those modifications available to the community.

It is _infuriating_ to me to think that there are versions of AzuraCast out there that have so many of our little nagging bugs fixed, that have better scheduling or better playlist management or more features, but rather than get pull requests submitted back to me, they're kept a secret to give one hosting company a competitive edge. That is not why I got into this business, and if I think about it too much, it makes me want to just stop doing the work entirely. I built AzuraCast so all the people who wanted to run self-hosted radio stations would be on a level playing field, and would all have access to the same powerful tools regardless of funds or technical ability. I did _not_ build AzuraCast so it could make other people money. I didn't build it so I could struggle despite the clear commercial utility of my own software.

I came from a fandom community that, for all its faults, was a friendlier place than the greater Internet radio ecosystem can be sometimes. When I got into this, I didn't realize how absolutely cutthroat and exploitative of a business it is. I still believe in what I do, but from what I've learned with my time in AzuraCast, I would never open-source my next project.

**Reply by johnrisby** [reply]
Thanks for the reply Buster. It's very disappointing to read that. If you can recommend any good hosting companies that do support the project, that'd be great (sorry, if they are listed on your site etc I'll find them, I've not had a chance to look yet).

I've also emailed you personally. If you have time to reply, that'd be great but I understand if not. Many thanks, and keep up the good work on a great project.
